sfcli/README.md

```diff
@@ -182,81 +162,76 @@ Options:
 
 Example:
 ```sh
-sfcli --url http://localhost:8099 watchlistmember registerFromDir -w:fingera -d:"d:\Fingera Registration Photo"
+sfcli --host localhost watchlistmember registerFromDir -w:fingera -d:"d:\Fingera Registration Photo"
 ```
 
-#### softrestart
+#### camera
 
-When config values are changed, they are not reflected immediately. To load changed configurations of you need to call softrestart command.
+This command allow to create, update or view settings of a camera. Camera is domain entity which groups some data entities, workers and configs. That allows you to add live stream from IP camera and easily change processing properties.
 
-Example:
-```sh
-$ sfcli --url http://localhost:8099 softrestart
-```
-
-#### video
-
-This command allow to create, update or view settings of videoProcessor. VideoProcessor is domain entity which groups some data entities, workers and configs. That alows you to add live stream from IP camera and easily change processing properties.
-
-NOTE: stream id is used as unique identifier of videoProcessor
+NOTE: camera id is a unique identifier of a camera
 
 ```
 Options:
   -?|-h|--help  Show help information
 
 Commands:
-  add           Create new video
-  get           Read properties of video
-  set           Edit properties of video
-
+  add           Create new camera
+  get           Read properties of camera
+  set           Edit properties of a camera
 ```
-Example (create videoProcessor):
+Example (create camera):
 ```sh
-$ sfcli --url http://localhost:8099 video add -v:c:\temp\wafs.mp4
+$ sfcli --host localhost camera add -v:c:\temp\wafs.mp4
 ```
-Example (start videoProcessor):
+Example (start camera):
 ```sh
-$ sfcli --url http://localhost:8099 video set -e:true -s:1
+$ sfcli --host localhost camera set -e:true -s:2976ac85-f570-4bec-9561-ce5b2ec1d234
 ```
 
-##### video add
+##### camera add
 
-Create video processor. If scope id is not provided then new scope will be created.
+Create a camera. `--name` and `--videoSource` options are required.
 
 ```
 Options:
-  --scopeId              Scope where stream will be created. If empty, new scope will be created.
-  -v|--videoSource       Url to video E.g. rtsp://server.example.org:8080/test.sdp
-  -e|--enabled           Whether the stream is processed or not
-  -i|--minEyeDistance    Minimum count of pixels between eyes
-  -x|--maxEyeDistance    Maximum count of pixels between eyes
-  -d|--faceDiscovery     Time between face re-detections in milliseconds
-  -p|--mjpegPreviewPort  Port to processed stream MJPEG preview
-  -?|-h|--help           Show help information
+  -n|--name                        [Required] Name of the new camera.
+  -v|--videoSource                 [Required] Url to video E.g. rtsp://server.example.org:8080/test.sdp
```

Comment:
not  big of a deal but video has source property not videoSource

Reply:
we can change the option to -s|--source, I kept it so that the options don't change too much, but I changed a lot already, so might as well change this :)